packages/schematics/src/command-line/nx.ts

```diff
@@ -1,49 +1,125 @@
-}
+yargs
+  .usage('Usage: $0 <command> [options>')
+  .command(
+    'affected',
+    'Compute affected applications',
+    yargs =>
+      yargs
+        .positional('action', {
+          choices: ['apps', 'build', 'e2e', 'dep-graph']
+        })
+        .command('<action> -- <SHA1 SHA2>', 'Targets within a range of SHAs')
+        .command(
+          '<action> -- files="<file,list>"',
+          'Targets a comma delimited list of files'
+        )
+        .command('<action> -- --uncommitted', 'Targets uncommited changes')
+        .command('<action> -- --untracked', 'Targets untracked changes')
+        .demandCommand()
+        .command(
+          'dep-graph',
+          'Generate a graph showing links between targets',
+          yargs =>
+            yargs
+              .describe('file', 'output file (e.g. --file=.vis/output.json)')
+              .command('-- <SHA1 SHA2>', 'Targets within a range of SHAs')
+              .command(
+                '-- files="<file,list>"',
+                'Targets a comma delimited list of files'
+              )
+              .command('-- --uncommitted', 'Targets uncommited changes')
+              .command('-- --untracked', 'Targets untracked changes')
+              .demandCommand()
+              .demand('file')
+              .demand('output')
+              .choices('output', [
+                OutputType.json,
+                OutputType.dot,
+                OutputType.html
+              ]),
+          _ => {
+            generateGraph(yargsParser(args));
+          }
+        )
+        .demandCommand()
+        .describe('args', 'Prints the names of the affected apps')
+        .describe('build', 'Builds the affected apps')
+        .describe('e2e', 'Tests the affected apps')
+        .describe(
+          'dep-graph',
+          'Generate a graph showing links between apps and libs'
+        ),
+    _ => {
+      affected(args);
+    }
+  )
+  .command(
+    'format <write|check>',
+    'Format workspace',
+    yargs =>
+      yargs
+        .describe('write', 'Overwrite non-formatted files')
+        .describe('check', 'Check if files need formatting'),
+    _ => {
+      format(args);
+    }
+  )
+  .command(
+    // TODO: delete this after 1.0
+    'update <check|skip>',
+    'Migrate to the latest version of NX',
+    yargs =>
+      yargs
+        .describe('check', 'Check if there are migrations to run')
+        .describe('skip', 'Skip running the latest set of migrations'),
+    _ => {
+      update(args);
+    }
+  )
+  .alias('update', 'migrates') // TODO: Remove after 1.0
+  .command(
+    'lint [files..]',
+    'Line workspace or set of files',
+    yargs => yargs,
+    _ => {
+      lint();
+    }
+  )
+  .command(
+    'postinstall',
+    false,
+    yargs => yargs.help(false),
+    _ => {
+      patchNg();
+      update(['check']);
+    }
+  )
+  .command(
+    'workspace-schematic <name>',
+    'Generate a schematic `ng g <name>`',
```

Comment:
Maybe a bit more descriptive for this line?:

Generate a custom schematic that can be run via \`ng g \<name\>\`